client/my-sites/comments/comment/style.scss

```diff
@@ -89,6 +89,12 @@
 	width: calc(100% - 48px);
 }
 
+.comment__author-has-link {
+	position: relative;
+	top: 4px;
+	margin-right: 5px;
+}
```

Comment:
Quite a nitpick here, but we could achieve the same with a simple `margin-bottom: -4px`.

And, to be super extremely consistent, I've kept most of the metrics multiple of `4px`, in particular the margin between gridicons and labels is `4px`.

So, to drop this to a one liner:
```css
.comment__author-has-link {
  margin: 0 4px 4px 0;
}
```

---

One last thing: I know it's odd, but what about dropping the `author` from the class?
`comment__author-has-link` -> `comment__has-link`

It's less consistent because we are in the `CommentAuthor` component (which was a really bad naming choice on my part... maybe `CommentMeta` would be more appropriate?), but at the same time "author has link" could be misleading?

Reply:
Thanks! Updated in 27b95c7524869e53b8a4490757242c3ab4434764.

> One last thing: I know it's odd, but what about dropping the author from the class?
`comment__author-has-link` -> `comment__has-link`

While I agree that a name is somewhat misleading but I'd like to keep it as is. First for the sake of consistency, and secondly because it produces a lint error. If we really need to change it it might be better to rename the parent component instead.